pkg/plugin/conftest/plugin.go

```diff
@@ -393,7 +393,7 @@ func (p *plugin) ParseConfigAuditReportData(ctx starboard.PluginContext, logsRea
 		for _, warning := range cr.Warnings {
```

Comment:
probebly we need to remove this as well :
```
if cr.Successes > 0 {
			successesCount += cr.Successes
		}
```

Reply:
Good catch. I have removed it.